tensorboard/defs/defs.bzl

```diff
@@ -149,3 +149,29 @@ def tf_ng_module(assets = [], **kwargs):
         angular_assets = assets,
         **kwargs
     )
+
+def tf_res_digest_suffixer(name, resources, template, out):
+    """Query parameter suffixer for a resource in a template.
```

Comment:
Cool. I’m okay with a new binary and macro for this. The more we have,
the more I consider defining something like a custom Jinja context with
filters for inlining base64 images, computing file digests, transcluding
files, etc. That way we could have fewer ad hoc genrules everywhere.